sw/device/tests/power_virus_systemtest.c

```diff
@@ -366,6 +375,12 @@ static void configure_pinmux(void) {
   CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIob12,
                                         kTopEarlgreyPinmuxOutselI2c2Scl));
 
+  // Enable pull-ups for SPI_HOST_0/1 data pins to avoid floating inputs.
+  if (kDeviceType == kDeviceSimDV || kDeviceType == kDeviceSimVerilator) {
```

Comment:
Why only for the simulation?

Reply:
pull up/down pad attributes are not supported for FPGA pins (see [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/prim_xilinx/rtl/prim_xilinx_pad_attr.sv;drc=c7dc533d4d139ae3146c608d64e232a1e427f46d;l=17)); IIRC they are set in the xdc file, see [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/data/pins_cw310.xdc;drc=ae9a6baef1c1e73cc4c73ce33b8cf5ad8e40a26d;l=119)